app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<header class="header-cat">
+<nav class="category-nav">
+<i class="fas fa-bars"></i>
+<h1>Categories</h1>
+<i class="fas fa-search"></i>
+</nav>
+</header>
+<section class="category-container">
+<% if @categories.empty? %>
+<p>Please add categories to see here, currently there are no categories for the budget</p>
+<% else %>
+<% @categories.each do |category| %>
+    <article class="category-article">
+    <%= link_to category_purchases_path(category.id) do %>
+      <div class="article-inner">
+   <div class="icon">
+  <img src="<%= category.icon %>" class="category-icon" alt="<%= category.name %>" />
+</div>
```

Comment:
- [ ] In the live demo version, please fix the broken icon of the first category for a professional and well-presented live version of the app. 🙏

Reply:
- Hey, according to the requested change, please, the issue pointed is not yet resolved as the live demo version of your app still shows the broken image icon. please, try to fix it to ensure your deployed version shows the correct image icon to make it more professional. see the image below:
 
 
![image](https://github.com/leloufadel/Money-tracker-App/assets/59397159/72df1cb1-75a7-461f-a444-7e6fd9090e55)